share/instruments/instruments.xml

```diff
@@ -11684,6 +11684,14 @@
                   <singleNoteDynamics>0</singleNoteDynamics>
                   <transposeDiatonic>-7</transposeDiatonic>
                   <transposeChromatic>-12</transposeChromatic>
+                  <Channel name="slap">
+                        <!--MIDI: Bank 0, Prog 36; MS General: Slap Bass-->
+                        <program value="36"/> <!--Slap Bass 1-->
+                  </Channel>
+                  <Channel name="pop">
+                        <!--MIDI: Bank 0, Prog 37; MS General: Pop Bass-->
+                        <program value="37"/> <!--Slap Bass 2-->
+                  </Channel>
```

Comment:
This is wrong, accustic bass should not default to slap but to pizzicato
Whether that should be named ' pizzicato' or just fall back to 'normal', like fot ehe other bass guitars is another issue to consider.